keps/sig-api-machinery/NNNN-crd-validation-defaulting-conversion-expressions/README.md

```diff
@@ -444,6 +382,82 @@ How will UX be reviewed, and by whom?
 Consider including folks who also work outside the SIG or subproject.
 -->
 
+
+### Future Plan
+
+
+#### Defaulting
+
+The `x-kubernetes-default` extension will be used. The location of the defaulter
+determines the scope of defaulter.
+
+The 'field' specifies which field is to be defaulted using a field path that is
+relative to the scope root. The defaulter will only be run if 'field' is unset, 
+and the result of the expression must match the type of the field to be
+defaulted.
+
+```yaml
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+...
+  schema:
+    openAPIV3Schema:
+      type: object
+      properties:
+        spec:
+          x-kubernetes-default:
+		    - rule: "!has(spec.type) && spec.Type == 'LoadBalancer' ? 'ServiceExternalTrafficPolicy' : null"
+			  field: spec.externalTrafficPolicy
+```
+
+- Each defaulter may have multiple rules.
+
+- The 'field' is optional and defaults to the scope if not explicitly set.
+
+If the expression evaluates to null, the field is left unset.
+TODO: Any downsides to using null this way?
```

Comment:
I think the idea here was to make it possible to conditionally default some field value only is some condition is true, otherwise don't do any defaulting. But the specifics of how we do that is a bit of an incomplete thought here.

Let's drop this from the KEP for now since this has been moved to future work and is an incomplete thought. We can circle back when we get to defaulting.

Reply:
I'll clean this up in a separate PR actually, since this PR just moved the text.